docs/fr/mkdocs.yml

```
@@ -60,6 +60,8 @@ nav:
 - Tutoriel - Guide utilisateur:
   - tutorial/background-tasks.md
 - async.md
+- Deploiement:
```

```suggestion
- Déploiement:
```

In your other PRs you use `Déploiement`

---

Nice catch